src/pages/workspaces/workspace/jobHistory/CallTable.js

```diff
@@ -92,17 +92,15 @@ const CallTable = ({ namespace, name, submissionId, workflowId, callName, callOb
             cellRenderer: ({ rowIndex }) => {
               const { failures, shardIndex: index, attempt } = callObjects[rowIndex]
               const failureCount = _.size(failures)
-              return failures ? [
-                h(Link, {
-                  style: { marginLeft: '0.5rem' },
-                  onClick: () => setFailuresModalParams({ index, attempt, failures })
-                }, [
-                  div({ style: { display: 'flex', alignItems: 'center' } }, [
-                    icon('warning-standard', { size: 18, style: { color: colors.warning(), marginRight: '0.5rem' } }),
-                    `${failureCount} Message${failureCount > 1 ? 's' : ''}`
-                  ])
+              return !!failureCount && h(Link, {
```

Comment:
The `Link` was previously in an array, which causes this warning: `index.js:1 Warning: Each child in a list should have a unique "key" prop.`